ecosystem/sep-0008.md

```diff
@@ -0,0 +1,48 @@
+
+## Preamble
```

Comment:
This should be in `draft` folder as described in https://github.com/stellar/stellar-protocol#sep-proposals.